File: swift/iniciante/1014.swift
import Foundation

if let inputA = readLine(), let inputB = readLine(),
   let totalDistanceTraveled = Int(inputA),
   let totalFuelConsumed = Double(inputB.replacingOccurrences(of: ",", with: "."))
{
    let averageConsumption = Double(totalDistanceTraveled) / totalFuelConsumed

    print(String(format: "%.3f km/l", averageConsumption))
}

File: javascript/iniciante/1014.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
});

const values = [];

rl.on("line", (line) => {
  values.push(...line.trim().split(/\s+/).filter(Boolean));

  if (values.length >= 2) {
    const totalDistanceTraveled = Number(values[0]);
    const totalFuelConsumed = Number(values[1]);
    const averageConsumption = totalDistanceTraveled / totalFuelConsumed;
    console.log(`${averageConsumption.toFixed(3)} km/l`);
    rl.close();
  }
});
